config.yaml

```diff
@@ -45,3 +45,8 @@ options:
     description: |
       If this is set, update-status will check if this node is currently validating for this address.
       HINT: if this config parameter is set, the actions is-validating-this-era and is-validating-next-era can be used.
+  parachain-spec-url:
+    type: string
+    default: ""
+    description: |
+      If this is set, the chain spec will be downloaded from this URL and used as the value for the --chain argument of the Polkadot binary.
```

Comment:
Just one thought; some chains use the `--chain` argument for both spec file **and** binary file downloads. So if this is set to download the spec file, maybe the user will mistakenly remove the `--chain` argument and then f*ck up the other download?

An edge case but just though of it :thinking:

Reply:
Long term we should get rid of the binary download based on `--chain` as well. Is it the docker extract you are referring to @jakobilobi ?

Reply:
Yeah I meant Docker downloads, whoops! I don't think `--chain` is used for binary DL:s?

Reply:
It's true that `--chain` is used for docker extracts. So we need to continue supporting cases where docker extract is used together with spec file download. Long term, if Mantle is successful, we should consider running the docker images directly instead of extracting. Then we can just specify the docker image in config, it should be more generic and not have to use `--chain` for extraction.